cppcache/include/geode/CacheableBuiltins.hpp

```diff
@@ -490,32 +322,18 @@ _GEODE_CACHEABLE_KEY_TYPE_(char16_t, CacheableCharacter, 3);
 
 // Instantiations for array built-in Cacheables
 
-_GEODE_CACHEABLE_ARRAY_TYPE_DEF_(int8_t, CacheableBytes);
```

Comment:
Do we not need to add explicit instantiation statements for these types now? How does this effect documentation with Doxygen?

Reply:
Good question.  I'm not sure.